packages/react-native/src/components/QuilttConnector.tsx

```
@@ -83,18 +84,11 @@ export const handleOAuthUrl = (oauthUrl: URL | string | null | undefined) => {
     // Open the normalized URL
     Linking.openURL(normalizedUrl)
   } catch (error) {
+    errorReporter.notify(new Error('OAuth URL handling error'), { oauthUrl, error })
     console.error('OAuth URL handling error')
 
-    // Only try the fallback if oauthUrl is not null
-    if (oauthUrl != null) {
-      try {
-        const fallbackUrl = typeof oauthUrl === 'string' ? oauthUrl : oauthUrl.toString()
-        console.log('Attempting fallback OAuth opening')
-        Linking.openURL(fallbackUrl)
-      } catch (fallbackError) {
-        console.error('Fallback OAuth opening failed')
-      }
-    }
+    // Bubble up to the caller
+    throw error
```

@zubairaziz this is fine, right?

---

In a way, yes. This will bubble up and crash the app. We don't have a way to gracefully reset the error as of now.

But it is one step up from failing silently